.github/ISSUE_TEMPLATE/flaking-test.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Flaking Test
+about: Report flaky tests or jobs in Kubernetes CI
+labels: kind/flake
+
+---
+
+<!-- Please only use this template for submitting reports about flaky tests or jobs (pass or fail with no underlying change in code) in Kubernetes CI -->
+
+#### Which jobs are flaking:
+
+#### Which test(s) are flaking:
+
+#### Testgrid link:
```

Comment:
Let's be more opinionated about the triage dash and have a section for it here rather than just saying we'd appreciate it?